README.md

```diff
@@ -337,7 +337,7 @@ yarn ts-node scripts/trigger-webhook-proposal.ts [EVENT] [ID]
 - `EVENT`: webhook event name
 - `ID`: webhook ID
 
-See https://docs.snapshot.org/tools/webhooks for the list of available `event` type.
+See [https://docs.snapshot.org/tools/webhooks](Snapshot webhooks documentation) for the list of available `event` type.
```

Comment:
```suggestion
See [Snapshot webhooks documentation](https://docs.snapshot.org/tools/webhooks) for the list of available `event` type.
```

Reply:
Right now it looks like this 
![image](https://github.com/snapshot-labs/envelop/assets/15967809/bdc21190-0b3d-4bb9-916b-909f7e9b79b6)